app/Phragile/Task.php

```diff
@@ -0,0 +1,116 @@
+<?php
+namespace Phragile;
+
+class Task {
+	private $title;
+	private $priority;
+	private $points;
+	private $status;
+	private $closed;
+	private $id;
+	private $assigneePHID;
+	private $assigneeName;
+	private $cssClass;
+
+	function __construct($attributes)
+	{
+		$fields = ['title', 'priority', 'points', 'status', 'closed', 'id', 'assigneePHID'];
+		foreach ($fields as $field)
+		{
+			if (!key_exists($field, $attributes))
+			{
+				throw new \InvalidArgumentException("The $field field is missing.");
+			}
+
+			$this->$field = $attributes[$field];
```

Comment:
this could make it difficult to find where exactly value of `$this->title` or `$this->closed` is set. But as long as the class and its constructor are that compact, I wouldn't oppose to have it like this here.